lib/challenge.js

```diff
@@ -229,10 +257,10 @@ let nums = [
 ];
```

Comment:
Per the instructions What would you add here in order to return `myArray`? (resubmit not required)